src/contracts/KolibriToken/components/BalanceBanner.tsx

```diff
@@ -0,0 +1,144 @@
+
+    function updatePendingRewards() {
+        const getData = async () => {
+            const rewards = await estimatePendingRewards(tezosUrl, selectedParentHash);
+            setPendingRewards(rewards);
+        };
+
+        getData();
+    }
+
+    function onSyncWallet() {
+        dispatch(syncWalletThunk());
+        updatePendingRewards();
+    }
+
+    function formatAmount(amount): string {
+        const minDigits = Math.min(token.round || 0, 18);
+        const maxDigits = Math.min(token.precision || 0, 18);
+        return new BigNumber(amount)
+            .dividedBy(10 ** (token.scale || 0))
+            .toNumber()
+            .toLocaleString(undefined, { minimumFractionDigits: minDigits, maximumFractionDigits: maxDigits });
+    }
+
+    function harvestTrigger() {
+        dispatch(setModalOpen(true, 'KolibriHarvest'));
+    }
+
+    return (
+        <Container>
+            <TopRow isReady={isReady}>
+                <Breadcrumbs>{breadcrumbs}</Breadcrumbs>
+                <Update onClick={onSyncWallet} time={time} isReady={isReady} isWalletSyncing={isWalletSyncing} />
+            </TopRow>
+            <BottomRow isReady={isReady}>
+                <AddressInfo>
+                    <AddressTitle>{displayName}</AddressTitle>{' '}
+                    {token.helpLink && (
+                        <AddressInfoLink onClick={() => openLink(token.helpLink || '')}>
+                            Learn more
+                            <LinkIcon iconName="new-window" size={ms(0)} color="white" onClick={() => openLink(token.helpLink || '')} />
+                        </AddressInfoLink>
+                    )}
+                </AddressInfo>
+                <AddressInfo>
+                    <TezosAddress address={publicKeyHash} weight={100} color="white" text={publicKeyHash} size={ms(1.7)} shorten={true} />
+                    <Gap />
+                    <div style={{ marginLeft: 'auto' }}>
+                        <Column>
+                            <AmountView
+                                color="white"
+                                size={ms(4.5)}
+                                amount={balance}
+                                weight="light"
+                                symbol={token.symbol}
+                                showTooltip={true}
+                                scale={token.scale}
+                                precision={token.precision}
+                                round={token.round}
+                            />
+                            {pendingRewards !== '0.0' && (
+                                <div style={{ float: 'right' }}>
+                                    <span style={{ marginRight: '5px' }}>
+                                        {t('components.plenty.nouns.pending_rewards')}: {pendingRewards} kDAO
+                                    </span>
+                                    <SmallButton buttonTheme="primary" onClick={() => harvestTrigger()}>
```

Comment:
you can use just fn reference to onClick if not passing any props

Reply:
the syntax re-writer might be doing this